src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -1618,6 +1618,13 @@ void OrbitMainWindow::ShowWarningWithDontShowAgainCheckboxIfNeeded(
   message_box.exec();
 }
 
+void OrbitMainWindow::ShowWarning(std::string_view title, std::string_view text) {
+  QMessageBox message_box{QMessageBox::Icon::Warning, QString::fromStdString(std::string{title}),
+                          QString::fromStdString(std::string{text}),
+                          QMessageBox::StandardButton::Ok, this};
+  message_box.exec();
```

Comment:
For your information, if you want you can also use
```
QMessageBox::warning(this, QString::fromStdString(std::string{title}), QString::fromStdString(std::string{text}))
```

Reply:
ack. Code removed.